libs/coin-modules/coin-bitcoin/jest.config.js

```diff
@@ -22,4 +22,5 @@ module.exports = {
       { outputName: "bitcoin-sonar-executionTests-report.xml", reportedFilePath: "absolute" },
     ],
   ],
+  // setupFilesAfterEnv: ["@ledgerhq/disable-network-setup"],
```

Comment:
Doesn't work yet ?

Reply:
@Justkant yes, I have to create tickets so we can tackle where it's commented (it means there is either a network call being done or a conflict with msw)